src/pages/AdvancedSearch/ResultsBlock/ResultsBlock.tsx

```diff
@@ -85,6 +88,14 @@ const valueRenderer: ValueRenderer = (valueAsString, value, ...keyPath) => {
   ) {
     return <Status status={value}>{valueAsString}</Status>;
   }
+  // Display date values as tooltip with relative date.
+  if (typeof value === "string" && isValid(parseISO(value))) {
+    return (
+      <Tooltip message={new Date(value).toLocaleString()}>
+        {formatFriendlyDateToNow(value)}
+      </Tooltip>
```

Comment:
Seeing this in action, I wonder if we should make use of the space and always display both, maybe something like this:

<img width="395" alt="Screenshot 2023-08-28 at 11 23 47 am" src="https://github.com/canonical/juju-dashboard/assets/361637/f3f72152-aee3-450a-8816-22c1bfa30408">


```suggestion
      <>
        {value}{" "}
        <span className="u-text--muted">
          ({formatFriendlyDateToNow(value)})
        </span>
      </>
```

Reply:
Makes sense! I've made these changes in my latest commit.